src/types/item/HarvestedFrom.svelte

```diff
@@ -16,7 +16,10 @@ const harvestedFrom = (data.byType("terrain") as (Terrain | Furniture)[])
   .filter((ter) =>
     (ter.harvest_by_season ?? []).some((h) => {
       if (!h.id) return false;
-      const harvest = data.byId("harvest", h.id);
+      const harvest = data.modEnabled()
+        ? data.byIdMaybe("harvest", h.id)
+        : data.byId("harvest", h.id);
+      if (!harvest) return false;
```

Comment:
```suggestion
      const harvest = data.byIdMaybe("harvest", h.id);
      if (!harvest) return false;
```
i think it's ok not to throw an error here when this isn't found, even if there are no mods enabled.